src/configs/typescript.js

```diff
@@ -11,7 +11,16 @@ module.exports = {
       readonly: 'array',
     }],
     semi: 'off',
-    '@typescript-eslint/comma-dangle': ['error', 'always-multiline'],
+    '@typescript-eslint/comma-dangle': ['error', {
+      arrays: 'always-multiline',
+      objects: 'always-multiline',
+      imports: 'always-multiline',
+      exports: 'always-multiline',
+      functions: 'always-multiline',
+      enums: 'always-multiline',
+      generics: 'ignore',
```

Comment:
why have we ignored for generics ?

Reply:
I wrote the explanation into PR description: https://github.com/technology-studio/eslint-config-txo-typescript/pull/73#issue-1379451014